ipapython/certdb.py

```diff
@@ -365,7 +365,9 @@ def create_db(self, user=None, group=None, mode=None, backup=False):
                                  os.O_CREAT | os.O_WRONLY,
                                  pwdfilemode), 'w', closefd=True) as f:
                 f.write(ipautil.ipa_generate_password())
+                # flush and sync tempfile inode
                 f.flush()
+                os.fsync(f.fileno())
```

Comment:
@tiran I call `fsync` here because `f` doesn't have a name, although I didn't sync the directory inode as in: https://github.com/freeipa/freeipa/blob/f47d86c719aa67f62f3d0c54f5270fc0fc8d1393/ipapython/directivesetter.py#L57-L67